### ISSUE TITLE
Allow keybinding assignments to override predefined accelerators

### ISSUE THREAD

**Issue by guidomz** [gnome-terminator/terminator#937]
I want to assign moving the cursor to panels to keys that are somewhat aligned to the way vim does, for instance `Alt-H` would trigger a `go_left`. However, I can not assign `go_right` to `Alt-L` which opens the layout launcher window, which I never use. (Coupled with the complication that when I assign `go_right` to something else like `Alt-;`, my vim-nimble fingers would open to layout launcher window which would not easily be closed with a simple `Esc` or even with `Ctrl-W` and `Ctrl-Q` like other apps can, but with an `Alt-F4` instead or, gasp, use the mouse.) Would it be possible to make such and similar keys to be modifiable as well? I've tried hard-configuring this through the `~/.config/terminator/config` file but even that doesn't override the accelator.

**Comment by mattrose** [comment]
Preferences->keybindings will let you edit any keybindings

**Comment by guidomz** [comment]
> Preferences->keybindings will let you edit any keybindings

I've tried that even before I wrote this issue and terminator says "Duplicate key bindings are not allowed. Key binding `Alt-L` is already in use to trigger the `Open layout launcher window` action." and there's no way to re-assign or remove the key assignment from the open layout launcher window. Thus, Preferences-> keybindings cannot edit **any** keybindings, prompting me to resort to the config file. I hope this clarifies my issue if my previous explanation was ambiguous.
